controllers/error_controllers.js

```diff
@@ -1,26 +1,27 @@
 
 exports.badRoute = (req, res) => {
-    res.status(404).send({ message: "invalid url" });
+    res.status(404).send({ message: "Bad request" });
 };
 
-exports.handlePSQL400s = (err, req, res, next) =>{
-    if (err.code === '22P02'){
-        res.status(400).send({msg:'Bad Request'});
-    } else {
-        next(err);
-    }
-
-};
-
-
 exports.handleCustomErrors = (err, req, res, next) =>{
+  
     if (err.status && err.msg){
         res.status( err.status).send({msg: err.msg});
 
     }else {
         next(err);
     }
 };
+exports.handlePSQL400s = (err, req, res, next) =>{
+    if (err.code === '22P02'){
+        res.status(400).send({msg:'Bad Request'})
+    }else if (err.code === '23503') {
```

Comment:
Off the top of my head, i think this error code is for foreign key issues, so might be better as `not found`